### PR TITLE
News post for playtest-20150801

### PR DESCRIPTION
**Pull request by obrakmann**

### CODE REVIEW

**Comment by Mailaender** [comment]
:+1:

File: content/news/playtest-20150719.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+We are pleased to announce the second playtest on the road to our next stable release.  With this playtest, we have entered the feature freeze phase, so it already is pretty close to what the release will eventually look like. There are probably a number of bugs still lurking in the code, so it needs testing, testing and some more testing!
```
Comment (pchote) [inline content/news/playtest-20150719.md]: ", so it already is pretty close to what the release will eventually look like." is really hard to parse.  Could we streamline this a bit?

File: content/news/playtest-20150719.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Compared to the previous playtest, there have been a number of new features, fixes and other changes all over the place. Music playback has received quite a number of changes. Maps can now define a soundtrack, which defaults to a random track, and also separate soundtracks for victory and defeat which are played on the score screen.
```
Comment (pchote) [inline content/news/playtest-20150719.md]: This is a really long sentence.  How about "Maps now have much greater control over their music, including default (starting) music and custom victory and defeat tracks".

File: content/news/playtest-20150719.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+In the multi-player lobby, selecting a map which contains custom rules will issue a warning message in the chat window to alert the players to the fact.
+
+We have also added some features that people watching replays, spectators and game casters will benefit from. Range circles of all kinds are now visible not only to the owner of the unit or building, but to allies and spectators as well. You are now also able to hide the in-game UI elements with a single hotkey (Ctrl+Shift+H by default).
+
+In-game, a noteworthy change is that paratroopers can now be selected and given orders while they are still in the air. Spies have also received a major overhaul. They are now able to detect other enemy spies as well as cloaked units, and infiltrating enemy production buildings enables you to build veteran units. These changes make spies more useful overall on the battlefield.
```
Comment (pchote) [inline content/news/playtest-20150719.md]: All these paragraphs are a big wall of text, which we know turns off casual readers.  Can we defer some of these to the bullet points below?

File: content/news/playtest-20150719.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+We are pleased to announce the second playtest on the road to our next stable release.  With this playtest, we have entered the feature freeze phase, so it already is pretty close to what the release will eventually look like. There are probably a number of bugs still lurking in the code, so it needs testing, testing and some more testing!
+
+Compared to the previous playtest, there have been a number of new features, fixes and other changes all over the place. Music playback has received quite a number of changes. Maps can now define a soundtrack, which defaults to a random track, and also separate soundtracks for victory and defeat which are played on the score screen.
+
+In the multi-player lobby, selecting a map which contains custom rules will issue a warning message in the chat window to alert the players to the fact.
+
+We have also added some features that people watching replays, spectators and game casters will benefit from. Range circles of all kinds are now visible not only to the owner of the unit or building, but to allies and spectators as well. You are now also able to hide the in-game UI elements with a single hotkey (Ctrl+Shift+H by default).
```
Comment (pchote) [inline content/news/playtest-20150719.md]: Maybe replace the second and third sentences with a plug for http://www.gamereplays.org/openra/?

File: content/news/playtest-20150719.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+<div style="text-align:center" markdown="1">
+
+![Custom map rule warning](/images/news/20150719-cnc-lobby.png)
+
+The game will warn players in the lobby when the host selects a map with custom rules.
+
+![Veteran unit production after infiltration](/images/news/20150719-ra-infiltrate-production-bonus.png)
+
+Infiltrating an enemy production building will let you build units with increased veterancy,<br/>indicated by the veterancy symbols in the production palette.
```
Comment (pchote) [inline content/news/playtest-20150719.md]: Please add "with a spy" after production building for clarity.

**Comment by pchote** [comment]
The content is good, and the media in particular is great.  I've left a couple of inline comments suggesting how to make it more reader-friendly.

**Comment by obrakmann** [comment]
Updated

**Comment by obrakmann** [comment]
Updated, added stuff about the latest merges to the news post.

**Comment by pchote** [comment]
I suggest cutting down a bunch of wording to make this easier to read:

```diff
diff --git a/content/news/playtest-20150801.md b/content/news/playtest-20150801.md
index 328f733..f1f868e 100644
--- a/content/news/playtest-20150801.md
+++ b/content/news/playtest-20150801.md
@@ -1,25 +1,24 @@
-We are pleased to announce the second playtest on the road to our next stable release.  We have now entered the feature freeze phase of the development cycle. As such, this playtest is already pretty close to what the final release will eventually look like. There are probably a number of bugs still lurking in the code, so it needs testing, testing and some more testing!
+We are pleased to announce the second playtest on the road to our next stable release.  We have now entered feature freeze phase, which means that we have stopped adding new features and instead focus on fixing any bugs that you might find.  That means that we need your help with testing, testing and some more testing!  Please report any bugs and problems you encounter [on our bug tracker](http://bugs.openra.net). Thank you for helping to improve OpenRA!
 
-Compared to the previous playtest, a number of new features, fixes and other changes have been added all over the place.
+A number of new features, fixes, and performance improvements make this a significant improvement over the last playtest and the June release.
 
-In the past few weeks, we have addressed a few long-standing performance issues caused by the AI and harvester logic, which you will have known from the noticeable lag spikes during long games. As part of this work, the AI has become a lot smarter about base building, does not use dogs anymore, can use the paratroopers superpower more reliably, and harvesters are less likely to become stuck.
+We are happy to announce that we have finally addressed a few long-standing performance issues that caused significant lag spikes in the mid to late game.
+As part of this work, the AI has become a lot smarter about base building, uses support powers more reliably, and harvesters are less likely to become stuck.
 
-Music playback has received quite a lot of changes. Maps now have much greater control over music, including the ability to specify default (starting) music and custom victory and defeat tracks.
-
-In-game, the most noteworthy change is that spies in the Red Alert mod have received a major overhaul, making them much more useful overall. They are now able to detect other enemy spies as well as cloaked units, and infiltrating enemy production buildings enables you to build veteran units.
-
-Of interest for modders and mappers will be the addition of a number of new Lua API functions. Disguising spies, capturing and infiltrating units and buildings using scripting is now possible. Other newly available functions can be used to play arbitrary sound files, flashing the screen and controlling the new palette effect trait. These latter ones can, in combination, be used to simulate simple weather effects, giving maps a unique touch. For an example take a look at the Fort Lonestar map, which now is set during a heavy rainstorm. Another use case would be day/night cycles.
+Music playback has received quite a lot of changes. Map authors now have much greater control over music, including the ability to specify default (starting) music and custom victory and defeat tracks.
 
 Other changes include:
 
-* Paratroopers can now receive orders while they are still in the air.
-* Our single-player missions have been polished to provider smoother gameplay.
-* A new ultra-hard difficulty has been added to the Allies 05 mission.
-* Replays of missions are available again.
-* In the multiplayer lobby, the game will warn the players when the host selectes a map with custom rules.
-* Range circles are now visible to allied players and spectators as well.
+* Paratroopers can now be selected and given orders before they land.
+* The single-player missions have been polished, Allies 05 has received a new ultra-hard difficulty, and mission replays can again be viewed through the replay browser.
+* A warning message is now given when a server admin selects a map with custom rules.  This should help combat the small number of dishonest players who have been creating cheat maps (shame on you!).
+* Range circles are now visible to allied players and spectators.
 * Players can hide the in-game UI using a hotkey, bound to Ctrl+Shift+H by default.
-* The media from 'The First Decade' compilation can now be used to install game content.
+* We now support installing game content from 'The First Decade' DVD.
+* Spies in the Red Alert mod have received a major overhaul.
+
+
+Of interest for modders and mappers will be the addition of new Lua API functions. Disguising spies, capturing and infiltrating units and buildings using scripting is now possible. Other newly available functions can be used to play arbitrary sound files, flashing the screen and controlling the new palette effect trait.
 
 Of course we did not only add new things, but also fixed bugs and annoyances.  Here's a small sample:
 
@@ -28,35 +27,27 @@ Of course we did not only add new things, but also fixed bugs and annoyances.  H
 * The 'Options' button will no longer blink for no reason in skirmish maps.
 * The duration bar on chronoshifted units is no longer visible to enemy players.
 * The aircraft and helicopter altitude has been increased again.
-* A bug in the color picker widget, which caused it to stay blank, has been fixed.
 
 For all the gritty details, check out the [full changelog](http://changelog.openra.net).
 
 Grab the installer for this playtest [from our download page](/downloads/) and give it a try!
 
-To ensure that the final release is as enjoyable as possible, we encourage you to report any bugs and problems you encounter [on our bug tracker](http://bugs.openra.net). Thank you for helping to improve OpenRA!
-
 <div style="text-align:center" markdown="1">
 
-![Custom map rule warning](/images/news/20150801-cnc-lobby.png)
-
-The game will warn players in the lobby when the host selects a map with custom rules.
-
 ![Veteran unit production after infiltration](/images/news/20150801-ra-infiltrate-production-bonus.png)
 
-Infiltrating an enemy production building with a spy will let you build units with increased veterancy,<br/>indicated by the veterancy symbols in the production palette.
+Infiltrating an enemy production building with a spy will let you build veteran units.
 
 ![Observer range circles and hidden gui](/images/news/20150801-d2k-range-circles.png)
 
-Allies and spectators can now see the range circles from units and buildings.
+Allied players and spectators can now see the range circles from units and buildings.
 
 <iframe width="600" height="333" src="https://www.youtube-nocookie.com/embed/3ai-MeYDx5c?rel=0" frameborder="0" allowfullscreen></iframe>
 
-OpenRA has added functionality that allows simulating simple weather effects.
+New Lua API functions can be used to simulate simple weather effects.
 
 <iframe width="600" height="333" src="https://www.youtube-nocookie.com/embed/WoZoQILTaGk?rel=0" frameborder="0" allowfullscreen></iframe>
 
-Behind the scenes, the work on the Tiberian Sun mod made another step forwards.<br/>
-Here's footage from the first proper test game that we had recently.
+Work continues on our Tiberian Sun mod.  Check out these clips from a recent test game!
 
 </div>
```

**Comment by obrakmann** [comment]
Updated. I changed some of your suggestions slightly to get rid of the long sentences in the listing.

File: content/news/playtest-20150801.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
+We are pleased to announce the second playtest on the road to our next stable release.  We have now entered feature freeze phase, which means that we have stopped adding new features and instead focus on fixing any bugs that you might find.  That means that we need your help with testing, testing and some more testing!  Please report any bugs and problems you encounter [on our bug tracker](http://bugs.openra.net). Thank you for helping to improve OpenRA!
+
+A number of new features, fixes, and performance improvements make this a significant improvement over the last playtest and the June release.
+
+We are happy to announce that we have finally addressed a few long-standing performance issues that caused significant lag spikes in the mid to late game.
+As part of this work, the AI has become a lot smarter about base building, uses support powers more reliably, and harvesters are less likely to become stuck.
```
Comment (Unit158) [inline content/news/playtest-20150801.md]: Why is this on a new line?

File: content/news/playtest-20150801.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+* A warning message is now given when a server admin selects a map with custom rules.
+* Range circles are now visible to allied players and spectators.
+* Players can hide the in-game UI using a hotkey, bound to Ctrl+Shift+H by default.
+* We now support installing game content from 'The First Decade' DVD.
+
+Of interest for modders and mappers will be the addition of new Lua API functions. Disguising spies, capturing and infiltrating units and buildings using scripting is now possible. Other newly available functions can be used to play arbitrary sound files, flashing the screen and controlling the new palette effect trait.
+
+Of course we did not only add new things, but also fixed bugs and annoyances.  Here's a small sample:
+
+* A number of crashes have been resolved.
```
Comment (Unit158) [inline content/news/playtest-20150801.md]: You're giving a sample of fixed things, and you do not enumerate what you have fixed. Could we have a couple examples of which crashes are fixed?

File: content/news/playtest-20150801.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+Infiltrating an enemy production building with a spy will let you build veteran units.
+
+![Observer range circles and hidden gui](/images/news/20150801-d2k-range-circles.png)
+
+Allied players and spectators can now see the range circles from units and buildings.
+
+<iframe width="600" height="333" src="https://www.youtube-nocookie.com/embed/3ai-MeYDx5c?rel=0" frameborder="0" allowfullscreen></iframe>
+
+New Lua API functions can be used to simulate simple weather effects.
```
Comment (Unit158) [inline content/news/playtest-20150801.md]: They are lighting effects, weather hasn't really been implemented yet.

File: content/news/playtest-20150801.md (hunk 7 above)
Comment (obrakmann) [inline content/news/playtest-20150801.md]: That would be too verbose, IMO. Those interested can take a look at the full changelog to find out.

File: content/news/playtest-20150801.md (hunk 6 above)
Comment (obrakmann) [inline content/news/playtest-20150801.md]: It's just in the source, it isn't actually rendered by the browser. I'll remove it on the next update.

File: content/news/playtest-20150801.md (hunk 7 above)
Comment (Unit158) [inline content/news/playtest-20150801.md]: It makes no sense to say you're going to enumerate something and not do it. This needs to move somewhere else.

File: content/news/playtest-20150801.md (hunk 8 above)
Comment (pchote) [inline content/news/playtest-20150801.md]: There are also sound effects.  Together they simulate a thunderstorm.

File: content/news/playtest-20150801.md (hunk 7 above)
Comment (pchote) [inline content/news/playtest-20150801.md]: We're enumerating fixed bugs and annoyances.  Crashes are a class of fixed bugs.

Comment (penev92) [inline content/news/playtest-20150801.md]: We do this every time. Most of the players don't know or care about the crashes. And we usually won't be able to list even half of the fixes.

File: content/news/playtest-20150801.md (hunk 8 above)
Comment (Unit158) [inline content/news/playtest-20150801.md]: Ah, I didn't know sounds couldn't be played on demand :)

**Comment by Unit158** [comment]
I don't fully agree with the list, but :+1: anyway.

File: content/news/playtest-20150801.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
+We are pleased to announce the second playtest on the road to our next stable release.  We have now entered feature freeze phase, which means that we have stopped adding new features and instead focus on fixing any bugs that you might find.  That means that we need your help with testing, testing and some more testing!  Please report any bugs and problems you encounter [on our bug tracker](http://bugs.openra.net). Thank you for helping to improve OpenRA!
```
Comment (Unit158) [inline content/news/playtest-20150801.md]: a or the feature freeze phase

Comment (Unit158) [inline content/news/playtest-20150801.md]: Double spaces after period (find.  That)

File: content/news/playtest-20150801.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+
+A number of new features, fixes, and performance improvements make this a significant improvement over the last playtest and the June release.
+
+We are happy to announce that we have finally addressed a few long-standing performance issues that caused significant lag spikes in the mid to late game.
+As part of this work, the AI has become a lot smarter about base building, uses support powers more reliably, and harvesters are less likely to become stuck.
+
+Music playback has received quite a lot of changes. Map authors now have much greater control over music, including the ability to specify default (starting) music and custom victory and defeat tracks.
```
Comment (Unit158) [inline content/news/playtest-20150801.md]: music and custom victory and defeat

File: content/news/playtest-20150801.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+
+Other changes include:
+
+* Spies in the Red Alert mod have received a major overhaul, making them much more useful.
+* Paratroopers can now be selected and given orders before they land.
+* Two new single-player missions have been added, Soviets 04a and 04b.
+* Single-player missions have been polished, and Allies 05 has received a new ultra-hard difficulty.
+* Mission replays can again be viewed through the replay browser.
```
Comment (Unit158) [inline content/news/playtest-20150801.md]: Is this a new feature or a fix? If it is new, you should probably remove again; if not, move it vv

File: content/news/playtest-20150801.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+* A warning message is now given when a server admin selects a map with custom rules.
+* Range circles are now visible to allied players and spectators.
+* Players can hide the in-game UI using a hotkey, bound to Ctrl+Shift+H by default.
```
Comment (Unit158) [inline content/news/playtest-20150801.md]: The hotkey isn't essential to understanding the sentence, and thus it should be in parentheses.

File: content/news/playtest-20150801.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+* We now support installing game content from 'The First Decade' DVD.
```
Comment (Unit158) [inline content/news/playtest-20150801.md]: Single quotes -> Italics? Unless quotes are the style here.

File: content/news/playtest-20150801.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+
+Of interest for modders and mappers will be the addition of new Lua API functions. Disguising spies, capturing and infiltrating units and buildings using scripting is now possible. Other newly available functions can be used to play arbitrary sound files, flashing the screen and controlling the new palette effect trait.
```
Comment (Unit158) [inline content/news/playtest-20150801.md]: Disguising spies, capturing and infiltrating units and buildings

Comment (Unit158) [inline content/news/playtest-20150801.md]: flash the screen and control the new palette...

File: content/news/playtest-20150801.md (hunk 9 above)
Comment (obrakmann) [inline content/news/playtest-20150801.md]: No, HTML compresses multiple whitespace characters into a single space (even newlines etc.), so it doesn't matter for rendering at all. It just better for reading the source (look up [English punctuation](https://en.wikipedia.org/wiki/Space_(punctuation)#Spaces_between_sentences)).

File: content/news/playtest-20150801.md (hunk 14 above)
Comment (Unit158) [inline content/news/playtest-20150801.md]: capturing or infiltrating units and buildings

File: content/news/playtest-20150801.md (hunk 10 above)
Comment (Unit158) [inline content/news/playtest-20150801.md]: default music, custom victory and defeat tracks. This is hacky, and to get the proper sentence you'd need to reword this.

Comment (Unit158) [inline content/news/playtest-20150801.md]: the ability to specify default music as well as custom victory and defeat tracks. BAM!

**Comment by obrakmann** [comment]
Updated.

**Comment by Mailaender** [comment]
Superseded by https://github.com/OpenRA/OpenRAWeb/issues/250.
